js_modules/dagster-ui/packages/app-oss/src/typings.d.ts

```diff
@@ -55,3 +55,67 @@ declare namespace Intl {
     timeZones: string[];
   }
 }
+
```

Comment:
Are the changes in this file needed? Don't see where they're being used

Reply:
Yep, these are typings that were missing in the app-oss package that we have ui-core. It was surfacing type errors when importing PNG. I copied over the missing types